lib/src/impl_js/impl_js.utils.dart

```diff
@@ -99,6 +99,25 @@ Object _translateDomException(
       '"${e.name}", message: $message');
 }
 
+/// Convert [Error] to [UnknownError].
+/// dart2wasm throws _JavaScriptError, but _JavaScriptError is not exposed.
+///
+/// [1]: https://github.com/dart-lang/sdk/issues/55496
+/// [2]: https://api.dart.dev/stable/latest/dart-core/Error-class.html
+Object _translateJavaScriptException() {
+  return UnknownError();
+}
+
+/// Error class for handling JavaScriptError that occurred in package:webcrypto.
+class UnknownError extends Error {
+  UnknownError();
```

Comment:
Let's not have anyone else creating instances of this error.
```suggestion
  UnknownError._();
```